quiz/quiz-questions.js

```diff
@@ -194,6 +199,12 @@ const cssQuestions = [
     answers: ["rgba(white)", "rgb(255, 255, 255)", "#fff", "white"],
     correctAnswer: "rgba(white)",
   }
+  {
+    question:
+    "what CSS property is used to apply a background color to a given HTML element?",
+    answers: ["Color", "Border-color", "Backcolor", "Background-color"],
```

Comment:
These are great questions.  Because CSS properties are written in lower-case, would you please edit the answers to lower-case?  That would make this perfect.